ordered_product/tests/test_ordered_product_page.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+from client.models import Client
+from supplier_product.models import SupplierProduct
+from delivery_location.models import DeliveryLocation
+from ordered_product.models import OrderedProduct
+from django.contrib.auth.models import User
+
+
+USER_NAME = 'meitar'
+PASSWORD = 'pass1234'
+VALID_PRODUCT_NAME = 'Banana'
+INVALID_PRODUCT_NAME = 'Eggplant'
+USER_NAME = 'meitar'
+VALID_SUPPLIER_PRODUCT_ID = '1'
+INVALID_SUPPLIER_PRODUCT_ID = '0'
+VALID_DELIVERY_LOCATION_ID = '3'
+INVALID_DELIVERY_LOCATION_ID = '9797'
+VALID_QUANTITY = '5'
+ORDER_PRODUCT_ID = '1'
+
+
+class Test_order_list:
+    @pytest.mark.django_db()
+    def test_empty_orders_list(self, client):
```

Comment:
Is there a better name for the test? what is empty orders list, what should we expect out of it?